### ISSUE TITLE
SPACE not supported in "target"?

### ISSUE THREAD

**Issue by hzqst** [appleboy/scp-action#85]
```
tar all files into /tmp/5369[24](https://github.com/hzqst/sven-fallguys-private/runs/8106441837?check_suite_focus=true#step:7:25)919/LbwKzVuPGc.tar
scp file to server.
create folder ***
untar file LbwKzVuPGc.tar
error:  tar: Co-op: Not found in archive
tar: Dedicated: Not found in archive
tar: Server: Not found in archive
tar: Exiting with failure status due to previous errors
```

the target is something like :

`target: "/root/.steam/steamapps/common/Sven Co-op Dedicated Server/"`

**Comment by appleboy** [comment]
Please post your config.

**Comment by hzqst** [comment]
```
      - name: Send svencoop over SSH
        uses: appleboy/scp-action@master
        with:
          host: ${{ secrets.GAME_SERVER_HOST }}
          username: ${{ secrets.GAME_SERVER_SSH_USERNAME }}
          password: ${{ secrets.GAME_SERVER_SSH_PASSWORD }}
          port: ${{ secrets.GAME_SERVER_SSH_PORT }}
          source: "svencoop/*"
          target: "/root/.steam/steamapps/common/Sven Co-op Dedicated Server/"
          strip_components: 1
          overwrite: true
```

this is my config and it turned out error like:

```
tar all files into /tmp/5369[24](https://github.com/hzqst/sven-fallguys-private/runs/8106441837?check_suite_focus=true#step:7:25)919/LbwKzVuPGc.tar
scp file to server.
create folder ***
untar file LbwKzVuPGc.tar
error:  tar: Co-op: Not found in archive
tar: Dedicated: Not found in archive
tar: Server: Not found in archive
tar: Exiting with failure status due to previous errors
```

**Comment by appleboy** [comment]
Fixed in #110
